diff --git a/CMakeLists.txt b/CMakeLists.txt
index 97327cc..1e7c7c8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -78,7 +78,7 @@ set(BACKWARD_ENABLE $<TARGET_OBJECTS:backward_object> CACHE STRING
 ###############################################################################
 # BACKWARD LIBRARY (Includes backward.cpp)
 ###############################################################################
-option(BACKWARD_SHARED "Build dynamic backward-cpp shared lib" OFF)
+option(BACKWARD_SHARED "Build dynamic backward-cpp shared lib" ON)
 
 if(BACKWARD_SHARED)
     set(libtype SHARED)
